utils/docker-utils.sh

```diff
@@ -181,12 +182,24 @@ docker::deploy_service() {
         docker_compose_dev="-c ${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_DEV_FILE"
     fi
 
+    if [[ -n "${DOCKER_COMPOSE_DEV_MOUNT}" ]]; then
+        docker_compose_dev="$docker_compose_dev -c ${DOCKER_COMPOSE_PATH}/$DOCKER_COMPOSE_DEV_MOUNT"
+    fi
```

Comment:
Can we add a comment here ?

Reply:
Yes sure, done in [d6a6ac1](https://github.com/jembi/platform/pull/191/commits/d6a6ac163363b2bd066cf25e6f00ee0034836cce)